examples/trl/dpo.py

```diff
@@ -84,6 +87,7 @@ class ScriptArguments:
     seed: Optional[int] = field(
         default=0, metadata={"help": "Random seed that will be set at the beginning of training."}
     )
+    deepspeed: Optional[str] = field(default=None, metadata={"help": "the deepspeed json config file"})
 
 
 def get_stack_exchange_paired(
```

Comment:
even if I reduce num_proc to 2, I keep getting a segmentation fault. However the map continues afterward. Num_proc=1 seems to run fine.

Reply:
I set default to None and didnot find crash